react exercise/users-information/src/types/user.ts

```diff
@@ -0,0 +1,87 @@
+export interface User {
+  id: number;
```

Comment:
There are many types of information here which are not relevant  (although they are in the json).
You can still ignore them by not including them in the type.

Reply:
I used them since I saw that in the next page which details the information of each user ("About me section") much of this info were included

Reply:
It might be redundant but I didn't think throughly about the next page implementation, this is why I did it but I can remove it for now until I will be sure how I'm planning on implementing next page About me section if you think it is better handling in this case

Reply:
Not critical for the exercise